vite.config.ts

```diff
@@ -15,11 +15,6 @@ export default defineConfig({
     react(),
     qrcode(),
     compression({
-      algorithm: 'gzip',
-      exclude: [/\.(br)$/, /\.(gz)$/],
-    }),
-    compression({
-      algorithm: 'brotliCompress',
```

Comment:
The compression plugin already set the `gzip` and `brotli` algorithm as default ones